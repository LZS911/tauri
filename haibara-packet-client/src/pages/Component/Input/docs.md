# Generate Input page by gl-cli
