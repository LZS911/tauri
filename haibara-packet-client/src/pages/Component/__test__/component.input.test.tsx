import Input from '../Input';
import { render } from '@testing-library/react';
describe('test Component', () => {
  it('should render Input to match snapshot', () => {
    const { container } = render(<Input />);
    expect(container).toMatchSnapshot();
  });
});
