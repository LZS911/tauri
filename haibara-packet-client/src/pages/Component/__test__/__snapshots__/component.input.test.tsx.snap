// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Component should render Input to match snapshot 1`] = `
<div>
  <div>
    Input
  </div>
</div>
`;
